packages/utilities/src/dom.ts

```diff
@@ -1,257 +0,0 @@
-import { IRectangle } from './IRectangle';
```

Comment:
As with the other file move/split, can you add a placeholder which re-exports everything which was originally in this file?

Reply:
see previous.